apps/graphql/src/apps/itinerary/dataloaders/Itineraries.js

```diff
@@ -35,9 +41,8 @@ export const parseParameters = (
     ? input.itinerary.inboundDate
     : null;
 
-  const flyFrom = unmaskID(origin.ids).join();
-  const flyTo =
-    destination && destination.ids ? unmaskID(destination.ids).join() : null;
+  const flyFrom = origin.ids && origin.ids.join();
+  const flyTo = destination && destination.ids ? destination.ids.join() : null;
```

Comment:
Also, I think we should use optional chaining and nullish coaleshing, so something like: 

```
const destinationIds = destination?.ids;

const flyFrom = (origin.ids ?? []).join();
const flyTo = destinationIds != null ? destinationIds.join() : null;
```

Reply:
Done!